deepspeed/runtime/zero/parameter_offload.py

```diff
@@ -259,6 +259,9 @@ def get_param_coordinator(self, training):
 
         return self.param_coordinators[training]
 
+    def empty_partition_cache(self):
+        self.partition_all_parameters()
```

Comment:
Can we add a docstring for this similar to what's in `docs/code-docs/source/zero3.rst`, then I think we can add it directly on our read-the-docs site as well.